flowpipe/node.py

```diff
@@ -598,9 +598,27 @@ def post_deserialize(self, data):
         self.identifier = data["identifier"]
         self.metadata = data["metadata"]
         self.file_location = data["file_location"]
-        node = import_class(
+
+        # The function could either be a ray function or, if the function is
```

Comment:
I don't understand the reference to ray at this point. It could be any callable, couldn't it?

Reply:
Sorry, that was just a weird late-night typo, I simple mean function, nothing to do with rays at all